test-ts/end-to-end.test.ts

```diff
@@ -56,22 +59,29 @@ describe("e2e", () => {
     [depositor0] = await randomSigner(parseUnits("300"));
     [depositor1] = await randomSigner(parseUnits("300"));
     [depositor2] = await randomSigner(parseUnits("300"));
-
+    [voter] = await randomSigner(parseUnits("300"));
+    const accounts = await hre.kit.contracts.getAccounts();
+    await accounts.createAccount().sendAndWaitForReceipt({
+      from: voter.address,
+    });
     groups = [];
     groupAddresses = [];
     validators = [];
     validatorAddresses = [];
-    for (let i = 0; i < 3; i++) {
+    for (let i = 0; i < 10; i++) {
       const [group] = await randomSigner(parseUnits("11000"));
       groups.push(group);
-      groupAddresses.push(groups[i].address);
+      if (i < 3) {
+        groupAddresses.push(groups[i].address);
```

Comment:
If groupAddresses contains only subset of all group addresses we should probably consider changing the name of the variable.